en/02_Developer_Guides/00_Model/11_Scaffolding.md

```diff
@@ -83,6 +83,91 @@ You can also alter the fields of built-in and module `DataObject` classes by imp
 > [!NOTE]
 > `FormField` scaffolding takes [`$field_labels` config](#field-labels) into account as well.
 
+## Scaffolding for relations
+
+Form fields are also automatically scaffolded for `has_one`, `has_many`, and `many_many` relations. These have sensible default implementations, but you can customise what form field will be used for any given `DataObject` model.
+
+> [!NOTE]
+> Polymorphic `has_one` relations do not have scaffolded form fields. Usually these are managed via a `has_many` relation.
```

Comment:
What side is "the other side"? I think this makes it more confusing rather than clearer.
I'll try "which points at the `has_one` relation" instead and see what you think.